crates/oxc_linter/src/rules/jest/no_test_prefixes.rs

```diff
@@ -0,0 +1,149 @@
+use std::borrow::Borrow;
+
+use oxc_ast::{AstKind, ast::Expression};
+use oxc_diagnostics::{
+    miette::{self, Diagnostic},
+    thiserror::Error,
+};
+use oxc_macros::declare_oxc_lint;
+use oxc_span::{Span, GetSpan};
+
+use crate::{context::LintContext, rule::Rule, AstNode, jest_ast_util::{parse_jest_fn_call, ParsedJestFnCall, JestFnKind}, fixer::Fix};
+
+#[derive(Debug, Error, Diagnostic)]
+#[error("eslint(jest/no-test-prefixes): Use {0:?} instead.")]
+#[diagnostic(severity(warning))]
+struct NoTestPrefixesDiagnostic<'a>(&'a str, #[label] pub Span);
+
+#[derive(Debug, Default, Clone)]
+pub struct NoTestPrefixes;
+
+declare_oxc_lint!(
+    /// ### What it does
+    /// 
+    /// Require using `.only` and `.skip` over `f` and `x`.
+    ///
+    /// ### Why is this bad?
+    ///
+    /// Jest allows you to choose how you want to define focused and skipped tests, 
+    /// with multiple permutations for each:
+    /// - only & skip: it.only, test.only, describe.only, it.skip, test.skip, describe.skip.
+    /// - 'f' & 'x': fit, fdescribe, xit, xtest, xdescribe.
+    /// 
+    /// This rule enforces usages from the only & skip list.
+    ///
+    /// ### Example
+    /// ```javascript
+    /// fit('foo'); // invalid
+    /// fdescribe('foo'); // invalid
+    /// xit('foo'); // invalid
+    /// xtest('foo'); // invalid
+    /// xdescribe('foo'); // invalid
+    /// ```
+    NoTestPrefixes,
+    correctness
```

Comment:
This is a convention lint rule, I wonder where should I put this 🤔 restriction?

Reply:
Maybe from rulegen ?